<launch>
  <!-- 1. nodelet manager -->
  <node pkg="nodelet"
        type="nodelet"
        name="nodelet_manager"
        args="manager"
        output="screen"/>

  <!-- 2. depth image â†’ point cloud -->
  <node pkg="nodelet"
        type="nodelet"
        name="point_cloud_xyz"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager"
        output="screen">

    <remap from="image_rect"       to="/camera/depth/image_raw"/>
    <remap from="camera_info" to="/camera/depth/camera_info"/>
    <remap from="points"      to="/camera/depth/points"/>
  </node>

  <!--3 pub_pc-->

  <!--node pkg="iphone_slam" type="pub_pc.py" name="point_cloud_0_pub" output="screen" required="true">
  </node-->
  <!--node pkg="iphone_slam" type="pc_to_world.py" name="pointcloud_to_world" output="screen" required="true">
  </node-->
</launch>

